### PR TITLE
Add travel time reliability measures

### PR DESCRIPTION
**Pull request by gshivaraman**

### CODE REVIEW

**Comment by jrawbits** [comment]
Once you're deeper through testing, I would like to include a sample VERSPM model in this pull that uses the TTR module (as a base model standalone), and separately a set of scenarios for sensitivity and reasonableness checking.

File: sources/modules/VETravelPerformance/R/TravelTimeReliability.R [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+# #
+# #</doc>
+# 
+# 
+# #=================================
+# #Packages used in code development
+# #=================================
+# #Uncomment following lines during code development. Recomment when done.
+# # library(visioneval)
+# # library(stringr)
```
Comment (jrawbits) [inline sources/modules/VETravelPerformance/R/TravelTimeReliability.R:40]: For testing, you could add magrittr and dplyr here. You need them explicitly marked as dependencies (see below where you import the pipe operator).

File: sources/modules/VETravelPerformance/R/TravelTimeReliability.R [hunk 2]
```diff
@@ -0,0 +1,352 @@
...
+      UNITS = "ID",
+      PROHIBIT = "",
+      ISELEMENTOF = ""
+    ),
+    item(
+      NAME = "Level",
+      TABLE = "OtherOpsEffectiveness",
+      GROUP = "Global",
+      TYPE = "character",
+      UNITS = "category",
```
Comment (jrawbits) [inline sources/modules/VETravelPerformance/R/TravelTimeReliability.R:77]: You should remove specifications for input items you're not using.

**Comment by jrawbits** [comment]
Do you have a recommendation for how I should go about testing the Travel Time Reliability module? You didn't include a model that uses it, and we had talked about possibly using the VEScenario set up to create test cases...

**Comment by gshivaraman** [comment]
Based on our first discussion, TTR has been set up in two stages, Stage-1 is a duplication of the VERSM model and Stage-2 reads the outputs from VERSPM to calculate travel time index. I have tested adding the TravelTimeReliabiliy model as part of VERSPM's run_model script (attached) and it works. Effectively, replacing the existing run_model R script in VERSPM, running the VERSPM model will produce all the travel time reliability. 
[run_model.zip](https://github.com/VisionEval/VisionEval-Dev/files/5681928/run_model.zip)

**Comment by jrawbits** [comment]
Thanks Gayathri. I'll try it out soon.

**Comment by jrawbits** [comment]
The travel time reliability code is now fully integrated in development-next.

**Pull request closed without merging**